### PR TITLE
Issue 56

### LINKED ISSUE #56: [fix] Accessories Landing Page
https://www.airlessco.com/emea/en/products/accessories/

needs to be styled. might need a content change.

### PR DESCRIPTION
**Pull request by helms-charity**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #56 

Test URLs:
- Before: https://main--airlessco--hlxsites.hlx.page/emea/cz/products/accessories/
- After: https://issue-56--airlessco--hlxsites.hlx.page/emea/cz/products/accessories/

### CODE REVIEW

File: blocks/productcards/productcards.js [hunk 1]
```diff
@@ -23,4 +23,6 @@ export default function decorate(block) {
     const text = document.createTextNode(link.textContent);
     body.replaceChild(text, link);
   });
+  const accessoriesTitle = document.querySelector('h1');
```
Comment (bstopp) [inline blocks/productcards/productcards.js]: This breaks when the document is not the accessories card. Check the console log of this page:

https://issue-56--airlessco--hlxsites.hlx.page/na/en/products/

File: blocks/productcards/productcards.css [hunk 2]
```diff
@@ -9,6 +9,19 @@
...
+h1#accessories-title strong {
+  color: var(--airless-blue);
+}
+
+.productcards-container {
+  padding-bottom: 50px;
+}
+
+.productcards-container h2 {
+  font-weight: bold;
```
Comment (bstopp) [inline blocks/productcards/productcards.css:23]: why "bold" and not a numerical value?

**Review by bstopp — CHANGES_REQUESTED** [review]

**Comment by bstopp** [comment]
Also, the change breaks the product landing page's width setting - compare:

https://issue-56--airlessco--hlxsites.hlx.page/na/en/products/
https://www.airlessco.com/na/en/products/

File: blocks/productcards/productcards.css (hunk 2 above)
Comment (helms-charity) [reply blocks/productcards/productcards.css:23]: changed to 700

File: blocks/productcards/productcards.js (hunk 1 above)
Comment (helms-charity) [reply blocks/productcards/productcards.js]: Moved to accessories-details

**Review by bstopp — APPROVED** [review]

**Pull request merged**